packages/reports/src/Containers/open-positions.tsx

```diff
@@ -190,23 +191,23 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
     const { isDesktop } = useDevice();
     const previous_active_positions = usePrevious(active_positions);
     const contract_types = [
-        { text: localize('Options'), is_default: !is_multiplier && !is_accumulator },
-        { text: localize('Multipliers'), is_default: is_multiplier },
-        { text: localize('Accumulators'), is_default: is_accumulator },
+        { text: 'Options', is_default: !is_multiplier && !is_accumulator },
+        { text: 'Multipliers', is_default: is_multiplier },
+        { text: 'Accumulators', is_default: is_accumulator },
     ];
     const [contract_type_value, setContractTypeValue] = React.useState(
-        contract_types.find(type => type.is_default)?.text || localize('Options')
+        contract_types.find(type => type.is_default)?.text || 'Options'
     );
     const prev_contract_type_value = usePrevious(contract_type_value);
-    const accumulator_rates = [localize('All growth rates'), '1%', '2%', '3%', '4%', '5%'];
+    const accumulator_rates = ['All growth rates', '1%', '2%', '3%', '4%', '5%'];
     const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
     const prev_accumulator_rate = usePrevious(accumulator_rate);
     const is_accumulator_selected = contract_type_value === contract_types[2].text;
     const is_multiplier_selected = contract_type_value === contract_types[1].text;
     const contract_types_list = contract_types
-        .filter(contract_type => contract_type.text !== localize('Accumulators') || !hide_accu_in_dropdown)
-        .map(({ text }) => ({ text, value: text }));
-    const accumulators_rates_list = accumulator_rates.map(value => ({ text: value, value }));
+        .filter(contract_type => contract_type.text !== 'Accumulators' || !hide_accu_in_dropdown)
+        .map(({ text }) => ({ text: localize(text), value: text }));
```

Comment:
not sure this will work?

Reply:
Could you please check test link?

Reply:
Or the pinned screen recording. 'Options' and 'Growth Rate' are translated as on production, but without bugs)

Reply:
@kate-deriv the extraction of the texts to be translated doesn't work this way. We shouldn't be using a variable inside localize.

Reply:
![image](https://github.com/binary-com/deriv-app/assets/56330681/2b363b2d-ee10-4cdb-801e-3e3bf13390b5)
Please check the regex pattern here. Only the texts inside the localize and i18n_default_text are extracted out and pushed to crowdin.

Reply:
Hmmm, then why it's working?

Reply:
There are two parts to the translation. One is the extraction of texts to be translated and the other one is actual translation.
It's working because the texts for which we removed localize are already pushed to crowdin and we have translated texts. Once we merge this PR, the translation extraction action will remove these texts on crowdin.

Reply:
Aaaa, okay, got it, will rework solution, thank you